sdk/eventgrid/azure-messaging-eventgrid/src/test/java/com/azure/messaging/eventgrid/DeserializationTests.java

```diff
@@ -173,32 +168,10 @@ private static Stream<Arguments> getObjectsForRoundTrip() {
             Arguments.of(BinaryData.fromObject(1)),
             Arguments.of(BinaryData.fromObject("data")),
             Arguments.of(BinaryData.fromString("{\"data\":\"data\"}")),
-            Arguments.of(BinaryData.fromObject(null)),
```

Comment:
If this is a recorded test you may need to re-record this as the number of tests running here has changed which can mess up which recording Test Proxy will use for it